wfl/Database/IDBManager.cs

```diff
@@ -7,7 +7,9 @@ namespace Symphono.Wfl.Database
     public interface IDBManager
     {
         Task InsertRestaurantAsync(RestaurantDto r);
+        Task<bool> CheckRestaurantIdAsync(string id);
         Task<IEnumerable<RestaurantDto>> GetAllRestaurantsAsync();
+        Task<RestaurantDto> GetRestaurantWithIdAsync(string Id);
```

Comment:
I don't actually think it makes sense to use a RestaurantDto in this manner.  I realize some of us may not be on the same page, and that may be causing some confusion.  Sorry about that!  Once we return a RestaurantDto it defeats the purpose of some of the benefits we were going with to post a RestaurantDto (like not being able to provide an Id).  I'm good with this for now, but we should spend more time discussing potential alternatives.

I think once we discuss alternatives to `IDBManager` it'll make things a bit clearer for us.